src/sonic-yang-mgmt/sonic_yang_ext.py

```diff
@@ -591,18 +591,22 @@ def _xlateListInContainer(self, model, yang, configC, table, exceptionList):
     """
     def _xlateContainerInContainer(self, model, yang, configC, table):
         ccontainer = model
-        #print(ccontainer['@name'])
-        yang[ccontainer['@name']] = dict()
-        if not configC.get(ccontainer['@name']):
+        ccName = ccontainer['@name']
+        yang[ccName] = dict()
+        if ccName not in configC:
             return
-        self.sysLog(msg="xlateProcessListOfContainer: {}".format(ccontainer['@name']))
-        self._xlateContainer(ccontainer, yang[ccontainer['@name']], \
-        configC[ccontainer['@name']], table)
+        if len(ccName) == 0:
```

Comment:
if len(configC[ccName]) ==0: 

i.e if value is zero, delete the key. Here, delete the key means processing is done,

Reply:
Hi, I saw TACPLUS table already has configuration in sample_config_db.json.
I added a UT for field `global` with empty value.
From where I saw the issue, it failed at GCU test after other test configured TACPLUS table.
I also added a empty json apply GCU test https://github.com/Azure/sonic-mgmt/pull/5459  for simply checking the configDB.